colcon_devtools/verb/version_check.py

```diff
@@ -31,16 +32,25 @@ def main(self, *, context):  # noqa: D102
         for entry_point in colcon_extension_points.values():
             distributions.add(entry_point.dist)
 
-        pypi = ServerProxy('https://pypi.python.org/pypi')
+        base_url = 'https://pypi.python.org/pypi/{project}/json'
         for dist in sorted(distributions, key=lambda d: d.project_name):
-            versions = pypi.package_releases(dist.project_name)
-            if not versions:
+            url = base_url.format(project=dist.project_name)
+            response = urllib.request.urlopen(url)
+
+            if response.status != 200:
                 print(
                     '{dist.project_name}: could not find package on PyPI'
                     .format_map(locals()), file=sys.stderr)
                 continue
 
-            latest_version = versions[0]
+            try:
+                data = json.loads(response.read())
+                latest_version = data['info']['version']
+            except JSONDecodeError:
```

Comment:
Also `JSONDecodeError` is undefined here. Please run the proposed patch locally to make sure it actually works.